src/main/java/pl/javastart/task/PhoneContract.java

```diff
@@ -0,0 +1,59 @@
+package pl.javastart.task;
+
+public abstract class PhoneContract implements Phone {
+    private double accountBalance;
+    private double smsCost;
+    private double mmsCost;
+    private double callCost;
```

Comment:
To nie są cechy wspólne dla wszystkich umów. W umowie na abonament nie ma ani stanu konta, ani kosztu sms/mms/rozmów